app/logic/courseManagement.py

```diff
@@ -6,18 +6,18 @@
 from app.models.term import Term
 
 
-def pendingCourses(termId):
+def submittedCourses(termId):
     '''
     Queries the database to get all the neccessary information for
-    non approved and non completed courses.
+    non approved and non submitted courses.
```

Comment:
This comment isn't correct. The query is getting information for courses that are submitted (implied would be not yet approved)